### ISSUE TITLE
chimGroup.R Error in `$<-.data.frame`(`*tmp*`, "RMDB",

### ISSUE THREAD

**Issue by qoldt** [ababaian/LIONS#2]
Hi Artem,
When running Westlions.sh I'm getting a `data.frame` error in R.  It seems an extra row is being generated:

    Rscript chimGroup.R ../../input.list ../../rmStats.Rdata 1 1
    [1] " Running standard analysis"
    Error in `$<-.data.frame`(`*tmp*`, "RMDB", value = c(249320L, 302633L,  : 
    replacement has 9538 rows, data has 9537
    Calls: $<- -> $<-.data.frame
    In addition: Warning message:
    In `[<-.data.frame`(`*tmp*`, "Group", value = list(numeric(0), numeric(0),  :
    provided 9537 variables to replace 1 variables
    Execution halted

**Comment by ababaian** [comment]
Would you be able to email me the input data? I'd like to run this line-by-line to figure out where the error is arising.

**Comment by ababaian** [comment]
I guess I should clarify @qoldt, the two chimGroup.R input data (they are much smaller files)
`input.list`
`rmStats.Rdata`

**Comment by b12loroc** [comment]
Hi Artem, when running east lion with fastq file, exist an error with tophat, if i running lions with bam file it happens the same that @qoldt  to me

**Comment by ababaian** [comment]
I can't seem to replicate the error. As I'm running the script it goes through and produces the output successfully.

One thing I had to change though was making the `input.list` file tab-separated instead of space separated for it to be read. Can you try changing that and re-running?

**Comment by b12loroc** [comment]
i have already run lions with input.list file tab-separated and lions
didn´t detect paired fastq file, althought i separated input with comma.
 Running tophat2 ...
  cmd:
/mnt/home/users/pab_001_uma/b12loroc/Protocolo/Experimento/LIONS_171220/bin/tophat2
 -p 1 -r 50 --report-secondary-alignments -o
/mnt/home/users/pab_001_uma/b12loroc/Protocolo/Experimento/LIONS_171220/pr
ojects/sandboxadenocarcinoma/paciente1tumor hg38_illumina
/mnt/home/users/pab_001_uma/b12loroc/Protocolo/Experimento/LIONS_171220/projects/sandboxadenocarcinoma/paciente1tumor/temp.1.fq.gz
/mnt/home/users/pab_00
1_uma/b12loroc/Protocolo/Experimento/LIONS_171220/projects/sandboxadenocarcinoma/paciente1tumor/temp.2.fq.gz
  File "/mnt/backup/soft/tophat/programs/x86_64/2.0.13/tophat", line 1001
    except getopt.error, msg:
                       ^
SyntaxError: invalid syntax
du: cannot access `accepted_hits.bam': No such file or directory
/mnt/home/users/pab_001_uma/b12loroc/Protocolo/Experimento/LIONS_171220/scripts/eastLion.sh:
line 195: [: -ge: unary operator expected
Alignment probably didnt work
Cheers and thanks!

2018-03-01 17:10 GMT+01:00 Artem Babaian <notifications@github.com>:

**Comment by qoldt** [comment]
@b12loroc it looks like you have other issues. This thread is specifically about a problem in the westlion.sh script.

I deleted all the files, made sure `input.list` was tab delimited (using vi), and ran everything from the beginning again.  Still the problem remains. 

I looked at the relevant part of the Rscript and it looks good to me. 
Today I will try troubleshooting it locally. I'll let you know if I figure it out.

**Comment by ababaian** [comment]
@qoldt did you find a solution to this?

**Comment by qoldt** [comment]
I did find [this thread](https://stackoverflow.com/questions/29814912/error-replacement-has-x-rows-data-has-y) that dealt with the issue, but I couldn't resolve it in this context. For now I have written an R script specific to my experiment to filter the .lions file down to those that occur consistently in treatment conditions.  I may write to you later to inquire about custom settings (vs using screenTE or driverTE) as presently it is not picking up couple chimeras that I have found using a running window on the bam files in seqmonk.
